dali/test/python/test_video_pipeline.py

```diff
@@ -85,6 +96,14 @@ def test_stride_video_pipeline():
         pipe_out = pipe.run()
     del pipe
 
+def test_multiple_resolution_videopipeline():
+    pipe = VideoPipeRoot(batch_size=BATCH_SIZE, data=MUTLIPLE_RESOLUTION_ROOT)
+    pipe.build()
+    for i in range(ITER):
+        print("Iter " + str(i))
+        pipe_out = pipe.run()
+    del pipe
```

Comment:
is this line necessary?

Reply:
do you mean remove test_multiple_resolution_videopipeline or just remove iterations ?

Reply:
Thinking again about it, the test case will actually never fail. Except for videos with unusual resolutions such as 2x2 or extremely large resolutions. 
It can be removed in my opinion.

Reply:
@jantonguirao  Can you elaborate ?

Reply:
The question is if pipe need to be explicitly deleted or we can just relay on the fact it will be deleted when the variable goes out of scope.

Reply:
pipe should be deleted when it goes out of scope but i think its better to be explicit like the other tests in `test_video_pipeline.py`.